qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -114,13 +114,13 @@ def _test_delete(self):
         self._delete('/api/rgw/proxy/user', params={'uid': 'teuth-test-user'})
         self.assertStatus(200)
 
-        self._delete('/api/rgw/proxy/user', params={'uid': 'teuth-test-user'})
-        self.assertStatus(404)
-        resp = self._resp.json()
-        self.assertIn('Code', resp)
-        self.assertIn('HostId', resp)
-        self.assertIn('RequestId', resp)
-        self.assertEqual(resp['Code'], 'NoSuchUser')
+        resp = self._delete('/api/rgw/proxy/user', params={'uid': 'teuth-test-user'})
```

Comment:
`self._delete` does not return anything. Please add a line below with: `resp = self._resp.json()` as it was before this chage, or change the `_delete` method in `helper.py` to return the value.

Reply:
Sorry, you're right, ``self._delete`` does not return anything, but ``self._request``. Seems i did not realize the missing of the small word ``return`` :-)